packages/wallet-standard/src/wallet.ts

```diff
@@ -0,0 +1,287 @@
+import type { GlowAdapter, Network, PhantomAdapter, SolanaWindow } from '@glow-xyz/glow-client';
+import type {
+    SolanaSignAndSendTransactionFeature,
+    SolanaSignAndSendTransactionMethod,
+    SolanaSignAndSendTransactionOutput,
+    SolanaSignTransactionFeature,
+    SolanaSignTransactionMethod,
+    SolanaSignTransactionOutput,
+} from '@solana/wallet-standard-features';
+import type { Wallet } from '@wallet-standard/base';
+import type {
+    ConnectFeature,
+    ConnectMethod,
+    DisconnectFeature,
+    DisconnectMethod,
+    EventsFeature,
+    EventsListeners,
+    EventsNames,
+    EventsOnMethod,
+    SignMessageFeature,
+    SignMessageMethod,
+    SignMessageOutput,
+} from '@wallet-standard/features';
+import bs58 from 'bs58';
+import { Buffer } from 'buffer';
+import { GlowWalletAccount } from './account.js';
+import { icon } from './icon.js';
+import type { SolanaChain } from './solana.js';
+import { getNetworkForChain, isSolanaChain, SOLANA_CHAINS } from './solana.js';
+import { bytesEqual } from './util.js';
+
+declare const window: SolanaWindow;
+
+export type GlowFeature = {
+    'glow:': {
+        glow: GlowAdapter;
+        solana: PhantomAdapter;
```

Comment:
I'm going to remove this adapter since I think we don't need it w/ the new wallet standard?